msteams-platform/concepts/bots/bots-overview.md

```diff
@@ -1,37 +1,54 @@
+Build and connect intelligent bots to interact with Microsoft Teams users naturally through chat. Or provide a simple commands-based bot, to be used as your "command-line" interface for your broader Teams app experience. You can make a notification-only bot, which can push information relevant to your users directly to them in a channel or direct message. You can even bring your existing bot from another channel and add Teams-specific support to make your experience shine.
 
-If you are just looking for a way to simply extend your team by integrating with custom tools and services in a secure manner, check out our [custom bots](~/concepts/custom-bot) feature. 
+![Example of a bot assisting a user](~/assets/images/bot_example.png)
 
-## Bots in Microsoft Teams
+> [!TIP]
+> If you are just looking for a way to simply extend your team by integrating with custom tools and services in a secure manner, check out our [custom bots](#build-a-custom-bot) feature. Be aware, though, that custom bots simply leverage your existing web services&mdash;they can't access non-messaging APIs, perform asynchronous posting, or add button actions to cards.
 
->**Note:** At this time, Microsoft Teams bots support 1:1 chats and channel conversations. They do not yet support group chats. 
+## What you need to know: Bots
 
-Other than their hexagonal avatar icon, bots appear just like any other team member you interact with, in a team or in 1:1 conversation.  They are always online and do not have a mood message.
+Other than its hexagonal avatar icon, a bot appears just like any other team member you interact with, in a channel or in one-on-one conversations. It is always online and does not have a mood message.
 
-Bots in Teams can surface in a 1:1 context ("personal scope"), as a member of a team ("team scope"), or both. For the latter, they take part in a conversation only when you @mention them. For the former, you can address them via the conversation interface or access them in the apps personal experience from the app bar flyout.
+Bots in Teams can surface in a one-on-one context ("personal scope"), as a member of a team ("team scope"), or both. For the latter, they take part in a conversation only when you @mention them. For the former, you can address them via the conversation interface or access them in the apps personal experience from the app bar flyout.
 
 With Microsoft Teams apps, you can choose make the bot the star of your experience, or just a helper. Bots are distributed as part of your broader app package, which can include other capabilities such as tabs or compose extensions.
 
 If you bot is the star, be sure to take advantage of the [tabs](~/concepts/tabs/tabs-overview) capability as well. Use this rich web view to surface accompanying experiences and information that helps your users best interact with your service.
```

Comment:
I didn't quite get this? 'If you bot is the star'

Reply:
I interpret this to mean "If your bot is the primary experience offered by your app". (And that typo has been in the docs for an embarrassingly long time.)